api/envoy/config/trace/v3/skywalking.proto

```diff
@@ -0,0 +1,55 @@
+syntax = "proto3";
+
+package envoy.config.trace.v3;
+
+import "envoy/config/core/v3/grpc_service.proto";
+
+import "udpa/annotations/migrate.proto";
+import "udpa/annotations/status.proto";
+import "udpa/annotations/versioning.proto";
+import "validate/validate.proto";
+
+option java_package = "io.envoyproxy.envoy.config.trace.v3";
+option java_outer_classname = "SkywalkingProto";
+option java_multiple_files = true;
+option (udpa.annotations.file_migrate).move_to_package =
+    "envoy.extensions.tracers.skywalking.v4alpha";
+option (udpa.annotations.file_status).package_version_status = ACTIVE;
+
+// [#protodoc-title: SkyWalking tracer]
+
+// Configuration for the SkyWalking tracer.
+// [#extension: envoy.tracers.skywalking]
+message SkyWalkingConfig {
+  // SkyWalking collector service.
+  core.v3.GrpcService grpc_service = 1 [(validate.rules).message = {required: true}];
+
+  ClientConfig client_config = 2;
+}
+
+// Client config for SkyWalking tracer.
+// [#next-free-field: 6]
+message ClientConfig {
+  // Service name for SkyWalking tracer. If service_name is empty, then cluster
+  // name of Envoy will be used as service name.
+  string service_name = 1;
+
+  // Service instance name for SkyWalking tracer. If instance_name is empty, the
+  // node name of Envoy will be used as service instance name.
+  string instance_name = 2;
+
+  // Authentication token. Set client token if backend open token
+  // authentication.
+  string authentication = 3;
```

Comment:
use `SdsSecretConfig`?

Reply:
Thanks. SkyWalking use simple string as a  anthentication token. But I will check if SdsSecretConfig can cover it.

Reply:
It seems that the SdsSecretConfig and authentication fields solve different problems. 'authentication' is only used for identity authentication in a secure network environment. So here I just use a common string like SkyWalking agent. When we add TLS support later, a new field should be used.

Reply:
I think the description of this property is not clear, can you please update to clarify its specific intent. `authentication` is a very broad term so it deserves more context.

Reply:
@wbpcode the `SdsSecretConfig` provides a generic method for API to get sensitive data, this is what intended for the authentication token. It's not related to TLS or any other means. See how OAuth2 filter use this to deliver HMAC secret / token secret etc.

Reply:
@lizan Can we provide oneof `SdsSecretConfig` and `string`? And +1 what @basvanbeek said. I recommend to use `token`.